quixstreams/state/recovery.py

```diff
@@ -42,6 +42,11 @@ def __init__(
         self._changelog_lowwater: Optional[int] = None
         self._changelog_highwater: Optional[int] = None
         self._committed_offset = committed_offset
+        self._recovery_consume_position: Optional[int] = None
+        self._initial_offset: Optional[int] = None
+
+    def __str__(self):
+        return f"{self.changelog_name}[{self.partition_num}]"
```

Comment:
it's used in a couple places as debug, and just once for info logging (informing recovery is complete for said RP, which I think `info` is correct level there since actual recovery is generally rare).

Reply:
I think using `__repr__` here is more correct from the semantics point of view if it's used only for debugging.